File: scripts/msg_talker.py
import rospy
from beginner_tutorials.msg import vision_feedback

def talker():
    pub = rospy.Publisher('feedback', vision_feedback, queue_size=10)
    rospy.init_node('msg_talker', anonymous=True)
    rate = rospy.Rate(10) # 10hz
    while not rospy.is_shutdown():
        feedback_pose = vision_feedback()
        feedback_pose.x = 0.1
        feedback_pose.y = 0.2
        feedback_pose.theta = 0.3
        x_str = "feedback x = %f" % feedback_pose.x
        y_str = "feedback y = %f" % feedback_pose.y
        theta_str = "feedback theta = %f" % feedback_pose.theta
        rospy.loginfo(x_str)
        rospy.loginfo(y_str)
        rospy.loginfo(theta_str)
        pub.publish(feedback_pose)
        rate.sleep()

if __name__ == '__main__':
    try:
        talker()
    except rospy.ROSInterruptException:
        pass

File: src/msg_talker.cpp
#include "ros/ros.h"
#include "beginner_tutorials/vision_feedback.h"

int main(int argc, char **argv)
{
  ros::init(argc, argv, "msg_talker");
  ros::NodeHandle n;

  ros::Publisher feed_pose = n.advertise<beginner_tutorials::vision_feedback>("feedback", 1000);

  ros::Rate loop_rate(10);

  int count = 0;
  while (ros::ok())
  {
    beginner_tutorials::vision_feedback feedback;

    feedback.x = 0.1;
	feedback.y = 0.2;
	feedback.theta = 0.3;

    ROS_INFO("feedback x = %f", feedback.x);
	ROS_INFO("feedback y = %f", feedback.y);
	ROS_INFO("feedback theta = %f", feedback.theta);

    feed_pose.publish(feedback);

    ros::spinOnce();

    loop_rate.sleep();

    ++count;
  }
  return 0;
}